src/main/java/io/cdap/plugin/googleads/source/batch/GoogleAdsBatchSourceConfig.java

```diff
@@ -118,17 +123,15 @@ public GoogleAdsBatchSourceConfig(String referenceName) {
   @Name(REPORT_FIELDS)
   @Description("Fields")
```

Comment:
Add a better description for this and all other configurations. Should be mostly the same as in .md file.